_includes/articles/suredbits-enterprise-ln.md

```diff
@@ -0,0 +1,58 @@
+{:.post-meta}
```

Comment:
Perhaps we could ask Suredbits for an image to go with the field report? Chris's slide here: https://youtu.be/tbwy9mJIrZE?t=2929 for example?

Reply:
Ping @Christewart